docs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lightningjs/sdk",
-  "version": "5.1.0",
+  "version": "5.1.1",
```

Comment:
Let's make this 5.2.0 since we're adding a new feature (dynamic screen resolution)